src/client/client.ts

```diff
@@ -106,24 +106,32 @@ export default class TaskClient {
    * Initializes a task client using the provided account information,
    * creating the specified database and collection if necessary.
    *
-   * @param account         - Azure Cosmos DB account url
-   * @param database        - Azure Cosmos DB database name
-   * @param collection      - Azure Cosmos DB collection/container name
-   * @param aadCredentials  - Azure Identity Library TokenCredential
-   * @param options         - Client creation options
+   * @param subscriptionId      - Subscription ID of this cosmos DB
+   * @param resourceGroupName   - Resource Group of this cosmos DC
+   * @param account             - Azure Cosmos DB account url
+   * @param database            - Azure Cosmos DB database name
+   * @param collection          - Azure Cosmos DB collection/container name
+   * @param aadCredentials      - Azure Identity Library TokenCredential
+   * @param options             - Client creation options
    *
    * @returns Promise containing the initialized client
    *
    * @public
    */
   static async createFromCredential(
+    subscriptionId: string,
+    resourceGroupName: string,
+    accountEndpoint: string,
```

Comment:
what's the difference between `accountEndpoint` and `account`?

Reply:
...and can they be combined into a single variable?

Reply:
I thought they needed to be separate, upon further inspection I can likely format this based on the account name